README.md

```diff
@@ -3248,6 +3250,22 @@ the optional `replacer` and `space` are passed
 to
 [`JSON.stringify`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify).
 
+### Interop
+
+####  <a id="L-pointer"></a> [≡](#contents) [▶](https://calmm-js.github.io/partial.lenses/#L-pointer) [`L.pointer(jsonPointer) ~> lens`](#L-pointer "L.pointer: JSONPointer s a -> PLens s a") <small><sup>v11.21.0</sup></small>
+
+`L.pointer` converts a valid [JSON Pointer](https://tools.ietf.org/html/rfc6901) (string) into a bidirectional lens. Works with [JSON String](https://tools.ietf.org/html/rfc6901#section-5) and [URI Fragment Identifier](https://tools.ietf.org/html/rfc6901#section-6) representations.
+
+For Example:
+
+```js
+L.get(L.pointer("/foo/0"), {foo: [1, 2]})
+// 1
+L.modify(L.pointer("#/foo/1"), x => x + 1, {foo: [1, 2]})
+// {foo: [1, 3]}
+```
```

Comment:
Only the result from the last statement in a Klipse snippet is displayed.  To get both results shown one could have two snippets:

```js
L.get(L.pointer("/foo/0"), {foo: [1, 2]})
// 1
```
```js
L.modify(L.pointer("#/foo/1"), x => x + 1, {foo: [1, 2]})
// {foo: [1, 3]}
```

Unfortunately things get really slow with Klipse (BTW, Firefox is by far fastest at handling lots of Klipse snippets on a page) when there are lots of snippets.  As a temporary solution I've been thinking of merging adjacent snippets and using `log` to produce output:

```js
log( L.get(L.pointer("/foo/0"), {foo: [1, 2]}) )
// 1
log( L.modify(L.pointer("#/foo/1"), x => x + 1, {foo: [1, 2]}) )
// {foo: [1, 3]}
```

Reply:
Using `log` sounds good to me. I'll make the change.